config/game_options.txt

```diff
@@ -546,4 +546,19 @@ NEUTRAL_STATION_TRAITS 2.5 1
 NEGATIVE_STATION_TRAITS 0 8
 NEGATIVE_STATION_TRAITS 1 4
 NEGATIVE_STATION_TRAITS 2 2
-NEGATIVE_STATION_TRAITS 3 1
\ No newline at end of file
+NEGATIVE_STATION_TRAITS 3 1
+
+##------------------------------------------------
+
+# Configuration for character Quirks.
```

Comment:
get rid of this header

Reply:
Fixed by commit [a92046a](https://github.com/tgstation/tgstation/pull/81033/commits/a92046aa3bae9c31087217f30072e9a9c2be0d19)